src/lavinmq/mqtt/consts.cr

```diff
@@ -0,0 +1,7 @@
+module LavinMQ
+  module MQTT
+    EXCHANGE      = "mqtt.default"
```

Comment:
Me & @spuun discussed that we should have consts for every constant value we use more than once.
Having the consts.cr file makes it easier to avoid inconsistencies and find things. The plan is to introduce it for the amqp namespace aswell.